packages/next-on-pages/templates/cache/adaptor.ts

```diff
@@ -140,8 +151,35 @@ export class CacheAdaptor {
 
 	/**
 	 * Loads the tags manifest from the suspense cache.
+	 *
+	 * @param force Whether to force a reload of the tags manifest.
+	 */
+	public async loadTagsManifest(force = false): Promise<void> {
+		// Refresh tags manifest after 1 second or if not loaded.
+		const shouldLoad =
+			force ||
+			!this.tagsManifest ||
+			!this.lastTagsManifestLoad ||
+			Date.now() - this.lastTagsManifestLoad > 1000;
```

Comment:
Did you come up with the 1 second logic here? or is it something that Next.js/Vercel does?

Reply:
No, it isn't something I'm aware Next.js or Vercel does. I thought it could be a reasonable tradeoff, but I'm open for better ideas. The reason behind this is just to avoid re-loading the tags for each and every get, so, as an example, caching them just for the duration of the current request would be equally fine.

Reply:
Yeah, that's ok for me! 😄 

I think I'll have time to make this change this week. Thanks! 💯

Reply:
Awesome! thanks so very much! ❤️